extensions/virtuallistview/src/VirtualListCell.tsx

```diff
@@ -323,30 +340,48 @@ export class VirtualListCell<ItemInfo extends VirtualListCellInfo> extends RX.Co
 
         return (
             <RX.Animated.View
-                style={ [_styles.cellView, overflow, this._animatedStylePosition] }
+                style={ [_styles.cellView, overflow, this._animatedStylePosition, this._animatedStyleWidth] }
                 ref={ _virtualCellRef }
                 tabIndex={ this.props.tabIndex }
                 onLayout={ this.props.onLayout ? this._onLayout : undefined }
                 onFocus={ this.props.onCellFocus ? this._onFocus : undefined }
                 onBlur={ this.props.onCellFocus ? this._onBlur : undefined }
+                onPress={this.props.onItemSelected ? this._onPress : undefined}
```

Comment:
This onPress isn't going to properly handle gesture feedback on touch-based devices because you're not using an RX.Button here. With the old VLV, users would typically render an RX.Button if they wanted to make items pressable. By incorporating selection in the VLV, you're taking away this flexibility and removing some functionality in the process. You're also adding functionality, so maybe it's a good tradeoff, but it would be even better if you weren't taking away flexibility. For example, would it make sense to give the user a way to provide his own selection/focus capabilities, overriding the built-in behaviors?